graph.py

```diff
@@ -203,23 +211,27 @@ def other_end(self, vertex: Vertex) -> Vertex:
             'edge.other_end(vertex): vertex must be head or tail of edge')
 
     def incident(self, vertex: Vertex) -> bool:
-        """
-        Returns True iff the edge is incident with the
-        vertex.
-        :param vertex: The vertex
+        """Determine if this edge is incident with the specified vertex.
+
+        :param Vertex vertex: The vertex.
         :return: Whether the vertex is incident with the edge.
         """
+
         return self.head == vertex or self.tail == vertex
 
 
 class Graph(object):
-    def __init__(self, directed: bool, n: int=0, simple: bool=False, name: str='G'):
-        """
-        Creates a graph.
-        :param directed: Whether the graph should behave as a directed graph.
-        :param simple: Whether the graph should be a simple graph, that is, not have multi-edges or loops.
-        :param n: Optional, the number of vertices the graph should create immediately
+    disjoint_union_operator = '⊎'
+
+    def __init__(self, directed: bool, n: int = 0, simple: bool = False, name: str = 'G'):
```

Comment:
Had je directed niet eerder optional gemaakt?

Reply:
Ja, in een vorige versie van deze klasse. Voor nu maar niet gedaan. Maakt niet echt veel uit, behalve dat het nu wat boilerplate oplevert.